pyproject.toml

```diff
@@ -6,11 +6,18 @@ build-backend = "maturin"
 name = "navigator"
 requires-python = ">=3.7"
 classifiers = [
+    "Framework :: Robot Framework",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Software Development :: Embedded Systems",
     "Programming Language :: Rust",
-    "Programming Language :: Python :: Implementation :: CPython",
-    "Programming Language :: Python :: Implementation :: PyPy",
 ]
+description = "This package serves as the entry point for embedded applications using Python on Blue Robotics's Navigator"
+readme = "README.md"
 
+[project.urls]
+Homepage = "https://bluerobotics.com/store/comm-control-power/control/navigator/"
+Documentation = "https://docs.bluerobotics.com/navigator-lib/"
```

Comment:
can we put under `navigator-lib/python` ? to allow us in the future to have `navigator-lib/cpp` ?

Reply:
Did it, but in future would suggest to redirect to a common link, where we can put some common navigator descriptions, changelogs then redirect to the specific language targets (python, cpp and rust).

Also liked this getting start example from here:
-> https://pola-rs.github.io/polars-book/